### ISSUE TITLE
Project: Guest user overview in teams

### ISSUE THREAD

**Issue by Tanddant** [microsoft/hack-together-teams#69]
### Project name

Guest user overview

### Description

A fairly straight forward tool that let's admins get and overview of which guests are in their tenant, and from there do a couple of basics with that user:

- See last/recent sign ins
- Set contact info for Entra
- Block the user
- Delete the user
- Regenerate an invite link if the user has lost theirs
- See which groups (teams) the guest is a member of
- Links to Entra with both user and groups
- Get a quick overview based on simple to read color coding of which users aren't signing in

### Project Repository URL

https://github.com/Tanddant/TeamsGuestUserOverview

### Project video

https://www.youtube.com/watch?v=XvGfuqFo34s

### Team members

@Tanddant, @jeppesc11

**Comment by waldekmastykarz** [comment]
Awesome @Tanddant and @jeppesc11! Looking forward to checking it out! Meanwhile, would you both be willing to fill in a feedback survey? It would help us improve next time and we'd appreciate it very much: https://aka.ms/hack-together-teams/survey.

**Comment by Tanddant** [comment]
@waldekmastykarz absolutely! - feedback submitted, thanks for hosting an awesome event 🙌

**Comment by waldekmastykarz** [comment]
Thank you so much, and thank you for joining us and sharing your feedback ❤️

**Comment by jeppesc11** [comment]
Definitely! Just submitted :-)

*Med Venlig Hilsen*
Jeppe Spanggaard Christensen
Tlf.: 41 76 02 55


Den tirs. 13. jun. 2023 kl. 08.38 skrev Waldek Mastykarz <
***@***.***>:

**Comment by Tanddant** [comment]
@jeppesc11 Saying my code sucks is not feedback that useful to the team 😅
